src/components/SuperDrawer/SuperDrawer.tsx

```diff
@@ -65,6 +65,8 @@ export function SuperDrawer(): ReactPortal | null {
   const isDetail = currentContent !== firstContent;
   const title = content === undefined ? '' : currentContent.title || firstContent.title;
 
+  const width = !isDetail && firstContent && firstContent.width === "small" ? 560 : 1040;
```

Comment:
**request:** This makes me wonder what would happen if you opened a "Detail" page after the opening a small Drawer. Seems like if opening a "Detail" page (a sub page within the Drawer) we could end up changing the width from 560 to 1040. I think we'd be good to omit the `!isDetail` and _always_ determine the width based on the `firstContent`. Then a user doesn't have to also remember to set `width: "small"` when opening sub pages and it can just be based on the initial drawer content's size.

Reply:
About that, there are other `1040` hard-coded values in the `SuperDrawerContent` component
![imagen](https://user-images.githubusercontent.com/35903168/180859583-a5e64b9d-269c-4c3a-a597-a498ed5ed2ae.png)
Should we change those too?, I'm not familiar with this, but from what I can see from the Storybook test it is ok to leave those hard-coded

Reply:
Hey, sorry. Was in meetings. I believe those values are just for animations. It is telling it to grow from `0` to `1040` in 3ms. If we don't update it, then it'll still work fine, but the animation will really finish growing from 0 to 560 in ~1.5ms. It is not a big deal, but probably would be better for consistent animation timings if we updated these too. Thanks for catching!